### PR TITLE
replace role with clusterrole

### PR DESCRIPTION
**Pull request by ivaanko**
Hi!

Thanks for the helm chart though it didn't work out of the box on the cluster level. In this PR I basically copied rbac.yaml from deployment folder to the chart and helmified it a bit.

Thank you!
Ivan

### CODE REVIEW

**Comment by lwolf** [comment]
Thanks for the PR.
Will be great if you can add cluster-wide rbac alongside the one that are already there so people can chose.

**Comment by ivaanko** [comment]
Hi! Updated. Is it better now?

File: deploy/helm/kube-cleanup-operator/templates/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{{- if and .Values.rbac.create .Values.rbac.global }}
```
Comment (lwolf) [inline deploy/helm/kube-cleanup-operator/templates/rbac.yaml:1]: 1) you need to add similar double `if` to the current rbac files as well to check (rbac.create and not global) 
2) please split this file to have 1 object per file

Comment (ivaanko) [reply deploy/helm/kube-cleanup-operator/templates/rbac.yaml:1]: 1. sure, I thought about this but didn't do it to have things untouched. Will do.
2. I totally support this perfectionism, update is coming!

**Comment by ivaanko** [comment]
Updated and I can't wait to get rid of my tmp rbac fix in by clusters and start using your amazing chart out of the box!

**Comment by lwolf** [comment]
Merged and released. Thanks again for the PR 

```
$ helm repo update
$ helm search kube-cleanup
NAME                              	CHART VERSION	APP VERSION	DESCRIPTION
lwolf-charts/kube-cleanup-operator	1.0.1        	v0.8.1     	Kubernetes Operator to automatically delete completed Job...
```

**Pull request merged**